crates/optimism/flashblocks/src/worker.rs

```diff
@@ -125,8 +128,12 @@ where
             ExecutedBlock {
```

Comment:
I think it would be useful to add some additional constructors to ExecutedBlock, because this is currently initialized by builders as well, and having an easy way to migrate this breaking change would be ideal

Reply:
good point, @mattsse just to clarify do you mean constructors like 

```
    pub const fn new(
        recovered_block: Arc<RecoveredBlock<N::Block>>,
        execution_output: Arc<ExecutionOutcome<N::Receipt>>,
        trie_data: DeferredTrieData,
    ) -> Self {
        Self { recovered_block, execution_output, trie_data }
    }

    pub fn with_trie_data(
        recovered_block: Arc<RecoveredBlock<N::Block>>,
        execution_output: Arc<ExecutionOutcome<N::Receipt>>,
        trie_data: ComputedTrieData,
    ) -> Self {
        Self { recovered_block, execution_output, trie_data: DeferredTrieData::ready(trie_data) }
    }
```